cli/install/install.go

```diff
@@ -1061,6 +1057,24 @@ func installTarantoolEE(binDir string, includeDir string, installCtx InstallCtx,
 	return nil
 }
 
+// dirsIsWriteable checks if the current user has the write access to the passed directory.
+func dirsIsWriteable(dir string) bool {
+	return unix.Access(dir, unix.W_OK) == nil
+}
+
+// subDirIsWritable checks if the passed dir doesn't exist but can be created.
+func subDirIsWritable(dir string) bool {
+	var err error
+	for {
+		_, err = os.Stat(dir)
+		if os.IsNotExist(err) {
+			dir = filepath.Dir(dir)
+			continue
+		}
+		return dirsIsWriteable(dir)
```

Comment:
Why not using `filepath.Walk` ?

Reply:
Cause we need to walk up to the root, AFAIU walkdir works in another direction.

Reply:
accepted